src/openfl/display/BitmapData.hx

```diff
@@ -505,6 +505,17 @@ class BitmapData implements IBitmapDrawable {
 		
 		readable = false;
 		
```

Comment:
The purpose of `bitmapData.disposeImage` (I wonder if there's a better name?) is to mark an object as intended to be hardware only. The next time it becomes eligible to render, the software surface (stored in the Lime `image` object) is disposed, and only the hardware texture remains. This means that you can no longer `getPixel`, `setPixel` or run similar operations on the `BitmapData`, but it does save some memory. Disposing the hardware surface seems to go against this purpose?

Reply:
Oh, I thought it was a way to dispose the referenced texture. Then maybe the code should go elsewhere?